src/containerapp/HISTORY.rst

```diff
@@ -8,6 +8,7 @@ upcoming
 * 'az containerapp env create': Set identity only when `--mi-system-assigned` or `--mi-user-assigned` is specified.
 * 'az containerapp env create': Set identity only when `--system-assigned` or `--user-assigned` is specified.
 * 'az containerapp up': Support deploying Azure AI Foundry model to Container App with `--model-registry`, `--model-name`, `--model-version`.
+* 'az containerapp sessionpool update': Support `--mi-system-assigned`, `--mi-user-assigned` and `--registry-identity`
```

Comment:
What is the difference between parameters `--registry-identity` with `--mi-system-assigned` and `--mi-user-assigned`?

Reply:
--registry-identity is for pull-image from ACR registry
--mi-system-assigned and --mi-user-assigned are used by container, to call other azure service
In API view, --mi-system-assigned and --mi-user-assigned need set to managedIdentity lifecycle. 
--registry-identity is inside the registry credentials

Reply:
Got it, thanks